lib/ide/src/view/text_editor.rs

```diff
@@ -101,6 +106,15 @@ impl TextEditor {
                 text_editor.borrow().save();
             }
         });
+
+        let word_occurrence_keys = KeyMask::new_alt_character('j');
+        let text_editor          = Rc::downgrade(&self.rc);
+        keyboard_actions.set_action(word_occurrence_keys,move |_| {
+           if let Some(text_editor) = text_editor.upgrade() {
+               text_editor.borrow_mut().select_next_word_occurrence();
+           }
+        });
```

Comment:
There is set of predefined keyactions for each textfield, and I think selecting word occurrences should be there.
So move this code to lib/core/src/display/shape/text/text_field/frp/keyboard.rs (somewhere at line 154-156)